Add debugDir parameter to dynamic-simulation, dynamic-security-analysis, shortcircuit and ampl-executor (#3455)
powsybl/powsybl-core: docs/simulation/dynamic/configuration.md

**stopTime**  
`stopTime` defines when the simulation stops, in seconds. The default value of this property is `1`.

**debugDir**

> This property indicates a directory path where debug files will be dumped. If `null`, no file is dumped.

Reviewer: ```suggestion
This property specifies the directory path where debug files will be dumped. If `null`, no files will be dumped.
```
